extensions-core/namespace-lookup/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -275,11 +291,10 @@ public LookupExtractor get()
         public byte[] getCacheKey()
         {
           return ByteBuffer
-              .allocate(CLASS_CACHE_KEY.length + id.length + 1 + v.length + 1 + 1)
-              .put(id)
-              .put((byte) 0xFF)
-              .put(v)
-              .put((byte) 0xFF)
+              .allocate(CLASS_CACHE_KEY.length + 1 + id.length + 1 + v.length + 1 + 1)
+              .put(CLASS_CACHE_KEY).put((byte) 0xFF)
+              .put(id).put((byte) 0xFF)
```

Comment:
not sure it is the good formatting you have sometime one call after the `.` and sometimes two calls like `put`, `put`